react/src/base/inputs/SprkRadio/SprkRadioItem/SprkRadioItem.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import uniqueId from 'lodash/uniqueId';
+
+const SprkRadioItem = (props) => {
+  const {
+    children,
+    variant,
+    idString,
+    additionalClasses,
+    analyticsString,
+    radioAdditionalClasses,
+    labelAdditionalClasses,
+    name,
+    value,
+    disabled,
+    onChangeFunc,
+    id,
+    ariaDescribedBy,
+    ...rest
+  } = props;
+  const internalId = uniqueId('sprk-radio-');
+  return (
+    <div
+      className={classnames(
+        'sprk-b-SelectionContainer sprk-b-Radio',
+        additionalClasses,
+        {
+          'sprk-b-Radio--huge': variant === 'huge',
+        },
+      )}
+      data-analytics={analyticsString}
+      data-id={idString}
+    >
+      <input
+        aria-describedby={ariaDescribedBy}
+        className={classnames('sprk-b-Radio__input', radioAdditionalClasses)}
+        data-id={idString}
+        disabled={disabled}
+        id={id || internalId}
+        name={name}
+        onChange={onChangeFunc}
+        type="radio"
+        value={value}
+        {...rest}
+      />
+      <label
+        className={classnames(
+          'sprk-b-Label sprk-b-Label--inline sprk-b-Radio__label',
+          labelAdditionalClasses,
+          {
+            'sprk-b-Label--disabled': disabled,
+          },
+        )}
+        htmlFor={id || internalId}
+      >
+        {children}
+      </label>
+    </div>
+  );
+};
+
+SprkRadioItem.propTypes = {
+  children: PropTypes.node,
```

Comment:
Should we add the same description here for `children` as we do on the other components?